src/index.html

```diff
@@ -6,13 +6,56 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <div class="container__photo"></div>
+      <div class="container__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="container__code">Product code: 195434</div>
+      <div class="container__feedback">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <article>Reviews: 5</article>
+      </div>
+      <div class="container__price">
+        <div class="container__price container__price--price">Price:</div>
+        <div class="container__price container__price--sum">$2,199</div>
+      </div>
+      <a
+        href="#"
+        data-qa="hover"
+        class="container__buttom"
+      >
+        Buy
+      </a>
```

Comment:
The 'Buy' button should be a `button` element instead of a link (`a` element). This is because the 'Buy' button does not navigate to a different page, it performs an action on the current page. A `button` element is more semantically correct in this case.